templates/traveling-page/traveling-page.css

```diff
@@ -0,0 +1,347 @@
+  font-size: 0.875rem;
+  text-align: left;
+  white-space: nowrap;
+}
+
+.city a {
+  text-decoration: underline;
+  color: var(--link-color-alt);
+  cursor: pointer;
+}
+
+.city a:hover {
+  text-decoration: none;
+}
+
+.city a[target="_blank"]::after {
+  content: "";
+  margin-left: 0.2em;
+}
+
+.section.city p:not(:last-child) {
+  margin-bottom: 1.25rem;
+}
+
+/* Designate areas */
+.section.city {
+  display: grid;
+  margin: auto;
+  padding-top: 4.rem;
+  grid-template-areas:
+    "name name"
+    "icons icons"
+    "links links"
+    "img1 img1"
+    "txt1 txt1"
+    "txt2 txt2"
+    "img2 img3"
+    "txt3 txt3"
+    "tips tips";
+}
+
+.section.city.backup-layout {
+  display: grid;
+  margin: auto;
+  padding-top: 4.rem;
+  grid-template-areas:
+    "name name"
+    "icons icons"
+    "links links"
+    "img1 img1"
+    "txt1 txt1"
+    "txt2 txt2"
+    "img2 img2"
+    "img3 img3"
+    "txt3 txt3"
+    "tips tips";
+}
+
+.section.city .default-content-wrapper {
+  grid-area: name;
+}
+
+.section.city .icon-type-wrapper {
+  grid-area: icons;
+}
+
+.section.city .link-box-wrapper {
+  grid-area: links;
+}
+
+.section.city .city-header-txt {
+  grid-area: txt1;
+}
+
+.section.city .city-header-img {
+  grid-area: img1;
+  height: 250px;
+}
+
+.section.city .city-middle-txt {
+  grid-area: txt2;
+}
+
+.section.city .city-middle-img {
+  grid-area: img2;
+}
+
+.section.city .city-footer-txt {
+  grid-area: txt3;
+}
+
+.section.city .city-footer-img {
+  grid-area: img3;
+}
+
+.section.city .tips-wrapper {
+  grid-area: tips;
+  width: 95%;
+}
+
+.section.city .city-header-txt,
+.section.city .city-middle-txt,
+.section.city .city-footer-txt,
+.section.city .city-header-img,
+.section.city .city-middle-img,
+.section.city .city-footer-img,
+.section.city .tips-wrapper {
+  margin-bottom: 1.8rem;
+}
+
+.section.city .city-middle-img img,
+.section.city .city-header-img img,
+.section.city .city-footer-img img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+
+@media (min-width:600px) {
+
+  .traveling-page .default-content-wrapper h1 {
+    font-size: 1.75rem;
+    margin-bottom: 2.5rem;
+  }
+
+  .traveling-page h2 {
+    font-size: 1.5rem;
+    margin: 1.75rem 0;
+  }
+
+  .section.city {
+    padding-top: 4rem;
+  }
+
+  .section.icon-type-container {
+    padding: 3rem 1rem;
+  }
+
+  .section.city.backup-layout {
+    column-gap: 2.2rem;
+    grid-template-areas:
+      "name name name name name name name name name name"
+      "icons icons icons icons icons icons icons icons icons icons"
+      "links links links links links links links links links links"
+      "img1 img1 img1 img1 img1 img1 img1 img1 img1 img1"
+      "txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1"
+      "txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 "
+      "img2 img2 img2 img2 txt3 txt3 txt3 txt3 txt3 txt3"
+      "img3 img3 img3 img3 txt3 txt3 txt3 txt3 txt3 txt3"
+      "tips tips tips tips tips tips tips tips tips tips";
```

Comment:
Could we simplify to?
```suggestion
      "name name"
      "icons icons"
      "links links"
      "img1 img1"
      "txt1 txt1"
      "txt2 txt2"
      "img2 txt3"
      "img3 txt3"
      "tips tips";
```
and potentially play with the column sizes as needed

Reply:
If you were suggesting the following code, I think because each item in a different row span differently, which means they need to be separately defined. The code becomes lengthy and not straightforward. 

```
img1 {
  grid-column: 1 / span 5;
  grid-row: 1;
}

name {
  grid-column: 6 / span 6;
  grid-row: 1;
}

icons {
  grid-column: 6 / span 6;
  grid-row: 2;
}

links {
  grid-column: 6 / span 6;
  grid-row: 3;
}

txt1 {
  grid-column: 6 / span 6;
  grid-row: 4;
}

txt2 {
  grid-column: 1 / span 5;
  grid-row: 5;
}

img2 {
  grid-column: 9 / span 4;
  grid-row: 5;
}

txt3 {
  grid-column: 5 / span 6;
  grid-row: 6;
}

img3 {
  grid-column: 1 / span 4;
  grid-row: 6;
}

tips {
  grid-column: 5 / span 7;
  grid-row: 7;
}
```


Do you think this will make it more readable and cleaner if I make all the area names aligned?
```   
grid-template-areas:
      "name name name name name name name name name name"
      "tags tags tags tags tags tags tags tags tags tags"
      "link link link link link link link link link link"
      "img1 img1 img1 img1 img1 img1 img1 img1 img1 img1"
      "txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1 txt1"
      "txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 txt2 "
      "img2 img2 img2 img2 txt3 txt3 txt3 txt3 txt3 txt3"
      "img3 img3 img3 img3 txt3 txt3 txt3 txt3 txt3 txt3"
      "tips tips tips tips tips tips tips tips tips tips";
```

Reply:
If you look at your layout, the columns do line up properly, so you can simplify to what I suggested above.
<img width="703" alt="Screenshot 2023-06-24 at 8 26 03 AM" src="https://github.com/hlxsites/petplace/assets/1235810/cf1aa8df-486e-453b-814a-10c08ba0ee66">

You can then use the column sizing, like:
```css
  grid-template:
      "name name"
      "icons icons"
      "links links"
      "img1 img1"
      "txt1 txt1"
      "txt2 txt2"
      "img2 txt3"
      "img3 txt3"
      "tips tips" / 1fr 2fr;
```
to have the grid lay all this out in a 33% / 66% size.

Reply:
Thank you. Solved!